src/getStepStartStates.ts

```diff
@@ -395,6 +404,21 @@ export default async function getStepStartStates(
     [STEP_PRIVATE_CA_CERTIFICATES]: createStepStartState(
       ServiceUsageName.PRIVATE_CA,
     ),
+    [STEP_BIG_TABLE_OPERATIONS]: createStepStartState(
+      ServiceUsageName.BIG_TABLE,
+    ),
+    [STEP_BIG_TABLE_INSTANCES]: createStepStartState(
+      ServiceUsageName.BIG_TABLE,
+    ),
+    [STEP_BIG_TABLE_APP_PROFILES]: createStepStartState(
+      ServiceUsageName.BIG_TABLE,
+    ),
+    [STEP_BIG_TABLE_CLUSTERS]: createStepStartState(ServiceUsageName.BIG_TABLE),
+    [STEP_BIG_TABLE_BACKUPS]: createStepStartState(ServiceUsageName.BIG_TABLE),
+    [STEP_BIG_TABLE_TABLES]: createStepStartState(ServiceUsageName.BIG_TABLE),
+    [STEP_BIG_TABLE_LOCATIONS]: createStepStartState(
+      ServiceUsageName.BIG_TABLE,
+    ),
```

Comment:
I've added a commit that removes the locations but forgot to update this file - please do this when you can.